cron/poll_read_issue_comments.py

```diff
@@ -0,0 +1,130 @@
+import logging
+import arrow
+import os
+from os.path import abspath, dirname
+
+import settings
+import github_api as gh
+
+THIS_DIR = dirname(abspath(__file__))
+# Hopefully this isn't overwritten on pulls
+SAVED_COMMANDS_FILE = join(THIS_DIR,'..',"data/issue_commands_ran.json")
+if not os.path.exists(SAVED_COMMANDS_FILE):
+    with open(SAVED_COMMANDS_FILE, 'w') as f:
+        dummy_data = {}
+        dummy_data["comment_ids_ran"] = []
+        json.dump(dummy_data, f)
+
+__log = logging.getLogger("chaosbot")
+
+'''
+Command Syntax
+/vote close closes issue when no nay reactions on this comment are added within voting window
+/vote reopen reopens issue when see above ^^^
+/vote label=<LABEL_TEXT> adds label when ^^^
+/vote remove-label=<LABEL_TEXT> removes label when ^^^
+/vote assign=<USER> assigns to user when ^^^
+/vote unassign=<USER> unassigns from user when ^^^
+'''
+
+COMMAND_LIST = ["/vote"]
+
+def can_run_vote_command(votes, comment_id):
+    json_data = {}
+    with open(SAVED_COMMANDS_FILE, 'r') as f:
+        json_data = json.load(f)
+        
+    # Already ran this command
+    if comment_id in json_data["comment_ids_ran"]:
+        return False
+    
+    # Voting window has passed
+    now = arrow.utcnow()
+    voting_window = gh.voting.get_voting_window(now)
+    
+    voting_window_over = gh.issues.is_issue_comment_in_voting_window(api, settings.URN, comment_id,
+                voting_window)
+    if not voting_window_over: # Still voting
+        return False
+
+    # At least one negative vote will cause vote to not pass
+    for user, vote in votes.items():
+        if vote < 0:
+            return False
+    
+    # Ugly putting it here, but it works
+    with open(SAVED_COMMANDS_FILE, 'w') as f:
+        json_data["comment_ids_ran"].append(comment_id)
+        json.dump(json_data, f)
+
+    return True
+
+def get_command_votes(api, urn, comment_id):
+    votes = {}
+    for voter, vote in gh.voting.get_comment_reaction_votes(api, urn, comment_id):
+        votes[voter] = vote
+    return votes
+
+def handle_vote_command(api, command, issue_id, comment_id, votes):
+    orig_command = command[:]
+    # Check for correct command syntax, ie, subcommands
+    log_warning = False
+    if(len(command)):
+        sub_command = command.pop(0)
+        if(sub_command == "close"):
+            if(can_run_vote_command(votes, comment_id)):
+                gh.issues.close_issue(api, settings.URN, issue_id)
+        elif(sub_command == "reopen"):
+            if(can_run_vote_command(votes, comment_id)):
+                gh.issues.open_issue(api, settings.URN, issue_id)
+        else:
+            # Other commands have an = in them
+            sub_command = sub_command.split("=")
+            if(len(sub_command > 1)):
+                args = sub_command[1:]
+                sub_command = sub_command[0]
+                if(sub_command == "label"):
```

Comment:
it is, but you still have to check which command you need to run. These commands would be implemented sometime in the future